homeassistant/components/solaredge/const.py

```diff
@@ -65,4 +66,15 @@
     "solar_power": ["PV", "Solar Power", None, "mdi:solar-power", False],
     "grid_power": ["GRID", "Grid Power", None, "mdi:power-plug", False],
     "storage_power": ["STORAGE", "Storage Power", None, "mdi:car-battery", False],
+    "purchased_power": ["Purchased", "Imported Power", None, "mdi:flash", False],
+    "production_power": ["Production", "Production Power", None, "mdi:flash", False],
+    "consumption_power": ["Consumption", "Cosumption Power", None, "mdi:flash", False],
+    "selfconsumption_power": [
+        "SelfConsumption",
+        "SelfConsumption Power",
```

Comment:
Is "selfconsumption" really a single word? Most search hits I see use a dash to bind self and consumption.

We normally don't use PascalCase for the entity name and for variable names and string constants in our api we use snake_case.

Reply:
Thanks for you review. I can agree with yours consideration and I've committed it. I've also fixed feedin_power. 
Anyway PR is now closed and I cannot push anymore. What do I do?

Reply:
Open a new PR :)

Reply:
> Open a new PR :)

Do you have any time to check it, please? :)